bee-api/bee-rest-api/src/endpoints/config.rs

```diff
@@ -213,7 +221,18 @@ impl RestApiConfig {
     }
 
     /// Returns the white flag solidification timeout.
-    pub fn white_flag_solidification_timeout(&self) -> u64 {
-        self.white_flag_solidification_timeout
+    pub fn white_flag_solidification_timeout(&self) -> &Duration {
```

Comment:
`Duration` implements `Copy` so I think it is better to return the value instead of a reference,

Reply:
done